index.html

```diff
@@ -6517,26 +6503,20 @@ <h3>Executing Script</h3>
 
  <dt>has an <a>own property</a> named "<code>toJSON</code>" that is
   a <a>Function</a>
- <dd>Return <a>success</a> with the value returned by
+ <dd><p><p>Let <var>result</var> be the return value of
```

Comment:
This isn't quite right, because `[[Call]]` will return a [completion record](https://tc39.es/ecma262/#sec-completion-record-specification-type) [evaluate](https://w3c.github.io/webdriver-bidi/#command-script-evaluate) in BiDi has a better example of how to run a script (and we probably do indeed need to do all the bookkeeping there). Altneratively you might be able to use https://w3c.github.io/webdriver/#dfn-execute-a-function-body although, I wonder if the alert handling there is what we want (a testcase with `{toJSON: () => alert("surprise!")}` would be informative).

Reply:
I didn't modify this content for that. Would it be ok for you when we move this particular request to a follow-up issue?

Reply:
This seems like one of the two fundamental spec bugs here. If you want to leave it to a followup, I'd avoid changing this entirely rather than changing it from one wrong thing to a different but equally wrong thing.